File: Competitive Programming/DAY-13/money.py
kara=int(input("Enter number of glasses kara sold"))
rani=int(input("Enter number of glasses rani sold"))

glassesk= kara*5
glassesr= rani*7

if glassesk > glassesr:
    print("Kara has more money by ",glassesk-glassesr," cents")
elif glassesr > glassesk:
    print("Rani has more money by ",glassesr-glassesk," cents")
else:
    print("Both have same amount of money ",glassesr," cents")

File: Competitive Programming/DAY-13/money.cpp
#include <iostream>

using namespace std;
void money(int k,int r)
{
    int kara= k*5;
    int rani=r*7;
    if(kara>rani)
    {
        cout<<"Kara has more money by "<<kara-rani<<" cents";
    }
    else if(kara<rani)
    {
        cout<<"Rani has more money by "<<rani-kara<<" cents";
    }
    else
    {
        cout<<"Both have same money "<<rani<<" cents";
    }
}

int main()
{
    int m,n;
    cout<<"Enter the number of glasses kara sold";
    cin>>m;
    cout<<"\nEnter the number of glasses rani sold";
    cin>>n;
    money(m,n);
    return 0;
}
